src/main/java/manufacturing/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -12,26 +12,25 @@
 import manufacturing.exception.DataProcessingException;
 import manufacturing.injections.Dao;
 import manufacturing.model.Manufacturer;
-import manufacturing.util.ConnectionController;
+import manufacturing.util.ConnectionUtil;
 
 @Dao
 public class ManufacturerDaoJdbc implements ManufacturerDao {
 
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        try (Connection connection = ConnectionController.getConnection()) {
-            String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
-            PreparedStatement insert =
-                    connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS);
-            insert.setString(1, manufacturer.getName());
-            insert.setString(2, manufacturer.getCountry());
-            insert.executeUpdate();
-            ResultSet resultSet = insert.getGeneratedKeys();
+        String query = "INSERT INTO manufacturers (name, country) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertStatement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+
```

Comment:
this empty line is redundant in all methods)

Reply:
I decided to do some separation for better readability